### PR TITLE
469: change drive indication start

### PR DESCRIPTION
**Pull request by sviatosluv**
ref: https://github.com/Buckitup-chat/chat/issues/469

log: [indicaton_log.txt](https://github.com/Buckitup-chat/platform/files/12520601/indicaton_log.txt)

### CODE REVIEW

File: lib/platform/storage/drive_indication_starter.ex [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+          next_supervisor: next_supervisor
+        } = state
+      ) do
+    DriveIndication.drive_init()
+    :timer.sleep(250)
+    DriveIndication.drive_reset()
+    :timer.sleep(250)
+    DriveIndication.drive_accepted()
+
+    Platform.start_next_stage(next_supervisor, next_specs)
```
Comment (sergey-lukianov) [inline lib/platform/storage/drive_indication_starter.ex:32]: we may start next stage as soon as we can

then `drive_init`
send after 250ms to drive_reset
which sends after 250ms to drive_accept

**Review by sergey-lukianov — CHANGES_REQUESTED** [review]

File: lib/platform/storage/drive_indication_starter.ex (hunk 1 above)
Comment (sviatosluv) [reply lib/platform/storage/drive_indication_starter.ex]: done

File: lib/platform/storage/drive_indication_starter.ex [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+  @impl true
+  def on_init(opts) do
+    next = opts |> Keyword.fetch!(:next)
+    next_specs = next |> Keyword.fetch!(:run)
+    next_supervisor = next |> Keyword.fetch!(:under)
+
+    Platform.start_next_stage(next_supervisor, next_specs)
+    DriveIndication.drive_init()
+    Process.send_after(self(), :reset, 250)
```
Comment (sergey-lukianov) [inline lib/platform/storage/drive_indication_starter.ex:15]: `Platform.start_next_stage` takes time , so should be after drive_init and send_after

**Review by sergey-lukianov — CHANGES_REQUESTED** [review]

File: lib/platform/storage/drive_indication_starter.ex (hunk 2 above)
Comment (sviatosluv) [reply lib/platform/storage/drive_indication_starter.ex]: done

**Review by sergey-lukianov — APPROVED** [review]

**Pull request merged**